specification/synapse/resource-manager/Microsoft.Synapse/preview/2020-04-01-preview/sqlPool.json

```diff
@@ -436,7 +436,7 @@
       ],
       "properties": {
         "sku": {
-          "$ref": "../../../../common/v1/types.json#/definitions/Sku",
```

Comment:
Can we remove the definitions/Sku object that is in this file now, to avoid confusion in future? https://github.com/Azure/azure-rest-api-specs/blob/master/specification/synapse/common/v1/types.json

Reply:
Removed